docs/pages/linking/into-your-app.mdx

```diff
@@ -0,0 +1,127 @@
+    '$ npx uri-scheme open myapp://somepath/details --ios',
+  ]}
+/>
+
+Running the above command:
+
+- Opens your app's `/details` screen
+- The `android` or `ios` options specify that the link should be opened on Android or iOS
+- Alternatively, you can open the link by searching for it in the device's native browser. For example, open Safari on iOS and type `myapp://` in the address bar to prompt the device to open your app.
+
+<Collapsible summary="Test a link using Expo Go">
+
+By default, [Expo Go](https://expo.dev/go) uses the `exp://` scheme. If you link to `exp://` without specifying a URL address afterward, it will open the app to the home screen. In development, your app's complete URL looks like `exp://127.0.0.1:8081`.
+
+To open the `/details` screen while testing on Expo Go, you can use `npx uri-scheme`:
+
+<Terminal
+  cmd={['$ npx uri-scheme open exp://127.0.0.1:8081/--/somepath/into/app?hello=world --ios']}
+/>
+
+In Expo Go, `/--/` is added to the URL when a path is specified. This indicates to Expo Go that the substring after it corresponds to the deep link path and is not part of the path to the app itself.
+
+By default, `exp://` is replaced with `http://` when opening a URL in Expo Go. You can also use `exps://` to open `https://` URLs. However, `exps://` does not currently support loading sites with insecure TLS certificates.
+
+</Collapsible>
+
+## Handle URLs
+
+You can observe links that launch your app using the [`Linking.useURL()`](/versions/latest/sdk/linking/#useurl) hook from [`expo-linking`](/versions/latest/sdk/linking/).
+
+```tsx index.tsx
+import * as Linking from 'expo-linking';
+
+export default function Home() {
+  const url = Linking.useURL();
+
+  return <Text>URL: {url}</Text>;
+}
+```
+
+The `Linking.useURL()` hook works behind the scenes by following these imperative methods:
+
+- The link that launched the app is initially returned using [`Linking.getInitialURL()`](/versions/latest/sdk/linking/#linkinggetinitialurl)
+- Any new links triggered while the app is already open are observed with [`Linking.addEventListener('url', callback)`](/versions/latest/sdk/linking/#linkingaddeventlistenertype-handler)
+
+## Parse URLs
+
+You can use the [`Linking.parse()`](/versions/latest/sdk/linking/#linkingparseurl) method to parse the **path**, **hostname**, and **query parameters** from a URL. This method extracts deep linking information and considers nonstandard implementations.
+
+```tsx index.tsx|collapseHeight=440
+import * as Linking from 'expo-linking';
+
+export default function Home() {
+  const url = Linking.useURL();
+
+  if (url) {
+    const { hostname, path, queryParams } = Linking.parse(url);
+
+    console.log(
+      `Linked to app with hostname: ${hostname}, path: ${path} and data: ${JSON.stringify(
+        queryParams
+      )}`
+    );
+  }
+
+  return (
+    /* @hide Your React component here. */ /* @end */
+  )
+}
+```
+
+## When to not use deep links
```

Comment:
Kind of goes for the entire page, but if we update the overview to clarify a link vs deep link, might be good to sync this language, because a lot of this is talking about links that may or may not be deep.